commands/operator-sdk/cmd/add/api.go

```diff
@@ -71,114 +70,43 @@ Example:
 }
 
 func apiRun(cmd *cobra.Command, args []string) {
-	projectPath := cmdutil.MustInProjectRoot()
-	fullProjectPath := mustGetwd()
-
 	// Create and validate new resource
 	r, err := scaffold.NewResource(apiVersion, kind)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// Must be new Kind: pkg/apis/<group>/<version>/<kind>_types.go shouldn't exist
-	typesFileName := r.LowerKind + "_types.go"
-	pkgApisDir := filepath.Join(fullProjectPath, "pkg", "apis", r.Group, r.Version)
-	mustNotExist(filepath.Join(pkgApisDir, typesFileName))
-
-	// TODO: Don't rewrite existing boilerplate files(register.go, doc.go) for new Kind
-	// scaffold pkg/apis/<group>/<version> directory
-	if err := os.MkdirAll(pkgApisDir, cmdutil.DefaultDirFileMode); err != nil {
-		log.Fatalf("failed to create %v: %v", pkgApisDir, err)
-	}
-
-	// scaffold pkg/apis/addtoscheme_<group>_<version>.go
-	fileName := "addtoscheme_" + r.Group + "_" + r.Version + ".go"
-	filePath := filepath.Join(fullProjectPath, "pkg", "apis", fileName)
-	codeGen := scaffold.NewAddToSchemeCodegen(&scaffold.AddToSchemeInput{ProjectPath: projectPath, Resource: r})
-	buf := &bytes.Buffer{}
-	if err := codeGen.Render(buf); err != nil {
-		log.Fatalf("failed to render the template for (%v): %v", filePath, err)
-	}
-	if err := writeFileAndPrint(filePath, buf.Bytes(), cmdutil.DefaultFileMode); err != nil {
-		log.Fatalf("failed to create %v: %v", filePath, err)
+	projectPath := cmdutil.MustGetwd()
```

Comment:
does `MustGetwd` return an absolute projectPath? if yes, it will be nice we name the variable `projectPath` to something like `fullProjectPath` or `absoluteProjectPath`.

Reply:
Yes it does. I will rename variables.